docs/error-codes.md

```diff
@@ -58,9 +58,9 @@ error and warning codes.
 
 - The value 'action' of the body attribute used in the substitution XML is invalid (the only supported options are `remove` and `stub`).
 
-#### `IL2012`: Could not find field 'field' for substitution
+#### `IL2012`: Could not find field 'field' in type 'type' specified in 'XML document location'
 
-- The field 'field' specified in the substitution XML was not found.
+- The XML descriptor preserves field 'field' on type 'type', but the field was not found.
```

Comment:
Same here - only the linktrim XML will "preserve" the field, the other XML formats do different things.